ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -0,0 +1,188 @@
+const RenewalCard = withStyle(renewalStyle.card)(
+  ({step, title, TitleComponent, lastCompletion, nextReview, children, style}) => {
+    const daysRemaining = daysFromNow(nextReview);
+    return <FlexColumn style={style}>
+      <div style={renewalStyle.h3}>STEP {step}</div>
+      <div style={renewalStyle.h3}><TitleComponent/></div>
+      <div style={{ color: colors.primary, margin: '0.5rem 0', display: 'grid', gridTemplateColumns: '6rem 1fr'}}>
+        <div>Last Updated On:</div>
+        <div>Next Review:</div>
+        <div>{`${displayDateWithoutHours(lastCompletion)}`}</div>
+        <div>{`${displayDateWithoutHours(nextReview)} (${daysRemaining > 0 ? daysRemaining + ' days' : 'expired'})`}</div>
+      </div>
+      {children}
+    </FlexColumn>;
+  }
+);
+
+const CompletedButton = ({buttonText, wasBypassed, style}:
+  {buttonText: string, wasBypassed: boolean, style?: React.CSSProperties}) => <Button disabled={true}
+                      data-test-id='completed-button'
+                      style={{
+                        height: '1.6rem',
+                        marginTop: 'auto',
+                        backgroundColor: colors.success,
+                        width: 'max-content',
+                        cursor: 'default',
+                        ...style
+                      }}>
+    <ClrIcon shape='check' style={{marginRight: '0.3rem'}}/>{wasBypassed ? 'Bypassed' : buttonText}
+  </Button>;
+
+const getExpirationTimeForModule = fp.curry((modules, moduleName) => {
+  return fp.flow(fp.find({moduleName: moduleName}), fp.get('expirationEpochMillis'))(modules);
+});
+
+const isComplete = (nextReview: number): boolean => daysFromNow(nextReview) > LOOKBACK_PERIOD;
+
+export const AccessRenewalPage = fp.flow(
+  withRouteData,
+  withProfileErrorModal
+)(() => {
+  const {profile: {
+    complianceTrainingCompletionTime,
+    dataUseAgreementCompletionTime,
+    publicationsLastConfirmedTime,
+    profileLastConfirmedTime,
+    dataUseAgreementBypassTime,
+    complianceTrainingBypassTime,
+    renewableAccessModules: {modules}},
+  } = useStore(profileStore);
+  const [publications, setPublications] = useState<boolean>(null);
+  const noReportId = useId();
+  const reportId = useId();
+  const getExpirationTimeFor = getExpirationTimeForModule(modules);
+
+  return <FadeBox style={{margin: '1rem auto 0', color: colors.primary}}>
+    <div style={{display: 'grid', gridTemplateColumns: '1.5rem 1fr', alignItems: 'center', columnGap: '.675rem'}}>
+      <BackArrow style={{height: '1.5rem', width: '1.5rem'}}/>
+      <div style={styles.h1}>Yearly Researcher Workbench access renewal</div>
+      <div style={{gridColumnStart: 2}}>Researchers are required to complete a number of steps as part of the annual renewal
+        to maintain access to <AoU/> data. Renewal of access will occur on a rolling basis annually (i.e. for each user, access
+        renewal will be due 365 days after the date of authorization to access <AoU/> data.
```

Comment:
nit: 365 would ideally be driven by the server config value

Reply:
Not seeing this in the config right now. I will revisit in RW-6777